src/content/docs/accounts/accounts/automated-user-management/roles-permissions-automated-user-management.mdx

```diff
@@ -9,7 +9,7 @@ redirects:
   - /docs/roles-permissions-automated-user-management
 ---
 
-import rolesimg26Permissions from 'images/Roles_26Permissions.png'
```

Comment:
@zuluecho9 there's a strange number of hyphens in that image name... but it might not matter.

Reply:
when you put a dash with space on either side in the image name field, it creates three dashes in file name